lib/PuppeteerSharp/Frame.cs

```diff
@@ -544,21 +544,40 @@ public Task<string> GetContentAsync()
                 if (document.documentElement)
                     retVal += document.documentElement.outerHTML;
                 return retVal;
-            }");
-
+            }");
+
         /// <summary>
         /// Sets the HTML markup to the page
         /// </summary>
-        /// <param name="html">HTML markup to assign to the page.</param>
+        /// <param name="html">HTML markup to assign to the page.</param>
+        /// <param name="options">The options</param>
         /// <returns>Task.</returns>
-        /// <seealso cref="Page.SetContentAsync(string)"/>
-        public Task SetContentAsync(string html)
-            => EvaluateFunctionAsync(@"html => {
+        /// <seealso cref="Page.SetContentAsync(string, NavigationOptions)"/>
+        public async Task SetContentAsync(string html, NavigationOptions options = null)
+        {
+            var waitUntil = options?.WaitUntil ?? new[] { WaitUntilNavigation.Load };
+            var timeout = options?.Timeout ?? 30_000;

```

Comment:
Let's move `30_000` to a const.

Reply:
we have a few places where we declare a default timeout of 30000
in `Page` & `FrameManager` and a few of the options classes
where should this constant be added to?